### PR TITLE
update for Shiny App

### PR DESCRIPTION
**Pull request by IhnwhiHeo**
@LaurentSmeets and @Rensvandeschoot 
This pull request implements suggestions @IhnwhiHeo made regarding the Shiny App. The only remaining thing is to adjust the plot used in the answer to the second question in Step 2. It turned out that the plot is imported from "plot.png" file, which I cannot find the original code.

### CODE REVIEW

**Comment by Rensvandeschoot** [comment]
thanks! Can you also upload the R file to generate the plot? and maybe create a subfolder 'www' with the correct files. This way, we have all files together for future updates.

File: PPS Shiny app/server.R [hunk 1]
```diff
@@ -47,17 +48,19 @@ server <- function(input, output, session) {
     showModal(modalDialog(
```
Comment (Rensvandeschoot) [inline PPS Shiny app/server.R:48]: can you add the reference inlclusing the zenode doi number in the app? not really sure what place it fits best, maybe at the top of the disclaimer?

File: PPS Shiny app/ui.R [hunk 2]
```diff
@@ -47,13 +48,12 @@ ui <- dashboardPage(
...
-                "PLoS One, 8(7): e68839. "))),
+                age on Ph.D. delay are estimated. Users learn about specifying a linear and a quadratic effect in the same model with the interactive visual plots, about how to think about plausible parameter spaces, and about 
+                the specification of normally distributed priors for regression coefficients."),
+              withMathJax(p("The data is based on data described in Van de Schoot, R., Yerkes, M.A., Mouw, J.M. & Sonneveld, H. (2013).", tags$a(href="https://www.rensvandeschoot.com/what-took-them-so-long-explaining-phd-delays-among-doctoral-candidates/", "What Took Them So Long? Explaining PhD Delays among Doctoral Candidates."), "PLoS One, 8(7): e68839.")),
+              withMathJax(p("You can cite the Ph.D. delay dataset in APA style as follows:", "Van de Schoot, R. (2020). PhD-delay Dataset for Online Stats Training [Data set]. Zenodo.", tags$a(href="https://doi.org/10.5281/zenodo.3999424", "https://doi.org/10.5281/zenodo.3999424")))),
```
Comment (Rensvandeschoot) [inline PPS Shiny app/ui.R:57]: nice! 
maybe also add the reference to the shiny app itself?

**Review by Rensvandeschoot — APPROVED** [review]

**Comment by IhnwhiHeo** [comment]
> thanks! Can you also upload the R file to generate the plot? and maybe create a subfolder 'www' with the correct files. This way, we have all files together for future updates.

@Rensvandeschoot 

This is indeed what we need to do! According to your review, I added a 'www' folder that contains necessary files to make Shiny App work and the reproducible R code to draw the 'NewPlot.png'.

File: PPS Shiny app/server.R (hunk 1 above)
Comment (IhnwhiHeo) [reply PPS Shiny app/server.R:48]: image


Agree with you! How about this? @Rensvandeschoot

Comment (Rensvandeschoot) [reply PPS Shiny app/server.R:48]: perferct!

**Pull request merged**